src/Form/Type/TranslatableChoiceType.php

```diff
@@ -37,12 +43,6 @@ class TranslatableChoiceType extends AbstractType
      */
     public function __construct(TranslatorInterface $translator)
     {
-        @trigger_error(
```

Comment:
Why do we need to move this `trigger_error`?

Reply:
It has been moved to head of the file because error should be thrown on file inclusion but not on object creation

Reply:
I am not sure of this change, because we are not super consistent with where we put those `trigger_error`. Normally we do it in the `__construct`. Would like to hear the opinion of @greg0ire on this also

Reply:
I've seen it in the header too (Admin.php)

Reply:
Class inheritance also should trigger error, so function call should be placed in header.